src/gb/gb.c

```diff
@@ -565,6 +566,11 @@ void GBSkipBIOS(struct GB* gb) {
 			GBVideoDisableCGB(&gb->video);
 		}
 		nextDiv = 0xC;
+		size_t i;
```

Comment:
I'm not sure this will compile everywhere since you're putting a variable declaration in the middle of a `switch` statement. That makes some compilers very unhappy.

Reply:
I can put it inside the for, but I saw that this was the style of this file.

Reply:
Also, I'm not sure about whether to initialize the wave RAM to the pattern always, or just in GCB and AGB.

Reply:
No, that's worse. You might need to declare it _outside_ of the switch, or put a block inside the case.

As for CGB and AGB vs always, I'd err towards always since that's the old behavior.

Reply:
In that case, I can just leave the loop outside of the switch for now.